src/pages/service.tsx

```diff
@@ -181,7 +186,7 @@ const ServicePage = ({ documentTypes }: { documentTypes: string[] }) => {
                 );
               }}
             >
-              {t('contribute/service:send_email')}
+              {t('service:send_email')}
```

Comment:
not sure what you do this now after you told me you wanted to do this way before :-)
TBD

Reply:
Their is not more reasons to do so with the sepatared projects.